.github/TUF_ON_CI_TEMPLATE/failure.md

```diff
@@ -0,0 +1 @@
+CC @sigstore/tuf-root-signing-staging-codeowners, please have a look.
```

Comment:
I kept it minimal for now but in future we could include like a mini-playbook here: how to deal with different workflow failures.